### ISSUE TITLE
BUG - Sev-3 - iOS - va.gov. announced as "V A dot governor"

### ISSUE THREAD

**Issue by alexandec** [department-of-veterans-affairs/va-mobile-app#8458]
## What happened?

VoiceOver on iOS announces the text "va.gov." as "V A dot governor". The period at the end causes VoiceOver to assume that "gov." is an abbreviation. This issue occurs in 7 translations across the app, based on a search for "va.gov." in common.json. The issue is not present on Android.

A potential fix is to modify the a11yLabelVA function to replace "va.gov." with "va.gov ." (adding a space before the period). Then we'd need to ensure that function is used to add an a11yLabel for every occurrence.

## Specs:

- Device: Any iPhone
- OS: iOS
- User Account: Any
- Accessibility State: VoiceOver on

## Steps to Reproduce

- Navigate to a screen where "va.gov." appears, for example Claims -> Remand return appeal
- Turn on VoiceOver
- Scroll down to the bottom so that "To review more details about your appeal, go to VA.gov." is announced

## Desired behavior

"VA.gov." should be read as "V A dot gov".

## Acceptance Criteria

VA.gov. is read correctly in all cases across the app.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [x] Desired behavior is added
- [x] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [x] Estimate of 1 added (for front-end tickets)
- [x] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by alexandec** [comment]
List of screens where the issue occurs:

- Claims -> Appeal Details -> Need Help
- Upocming appointment Details -> AppointmentAddressAndNumber (for example, Primary Care / John Smith / Cheyennne VA Medical Center in demo mode)
- Past appointment details (for example, Primary Care / John Smith / Cheyennne VA Medical Center in demo mode)
- Direct deposit
- Claim file upload -> Take Photos (2 occurrences on this screen)

**Comment by rbontrager** [comment]
@alexandec You missed an instance of VA.gov saying "V A dot governer" in the need to reschedule section for upcoming appointments. You can test in demo mode on the canceled appointment at albany VA Medical Center.

**Comment by rbontrager** [comment]
@alexandec You missed an instance of VA.gov saying "V A dot governer" in the need to reschedule section for upcoming appointments. You can test in demo mode on the canceled appointment at albany VA Medical Center.

**Comment by alexandec** [comment]
@rbontrager thanks, I made the fix

**Comment by rbontrager** [comment]
Verified that VA.gov is being announced via voiceOver in all instances.  Approved by QA.
